vue/components/forms/personalization-form/initials-images.vue

```diff
@@ -71,38 +75,82 @@ export const InitialsImages = {
             type: Boolean,
             default: false
         },
+        /**
+         * A function that receives the initials and engraving as strings
+         * and the img element that will be used and returns a map with
+         * the initials and a profile list.
+         */
         initialsBuilder: {
             type: Function,
             default: null
         },
+        /**
+         * The context to be used to display the image.
+         */
         context: {
             type: Array,
             default: () => ["step::personalization"]
         },
+        /**
+         * The viewport name to be used to showcase the initials.
+         */
         viewport: {
             type: String,
             default: null
         },
+        /**
+         * The function that gets the context to be used to display
+         * the image, in case the context is dynamic.
+         */
         getContext: {
             type: Function,
             default: null
         },
+        /**
+         * The frame to be shown in the images.
+         */
+        frame: {
+            type: String,
+            default: null
+        },
+        /**
+         * The max height of each image.
+         */
         imageHeight: {

```

Comment:
I know this was here already, but `imageHeight` and `height` are both related to the image, but one is for the `max-height`. Maybe renaming them to make the parameter's names clearer.